index.html

```diff
@@ -4922,10 +4922,7 @@ <h2><span class="chunk">fcTL</span> Frame Control Chunk</h2>
           step as long as the result is guaranteed to be identical. For example, if the default image is included in the animation,
           and uses a `blend_op` of `APNG_BLEND_OP_SOURCE`, clearing is not necessary because the entire output buffer will be
           overwritten.</p>
-          <!-- Not clear this para is needed, but the original specification had it -->
```

Comment:
Drive-by review: This comment still seems relevant. In particular "As noted earlier" followed by a normative requirement seems rather suspect. We shouldn't be requiring things twice. And the non-normative bits about optimizations might be better placed in a note.

Reply:
The comment relates to the following paragraph, not the preceding one.

The PR deletes the needless duplicate requirement (which is true of all chunks, this one is not special) and also the now-irrelevant comment.